scripts/west_commands/spdx.py

```diff
@@ -7,10 +7,11 @@
 
 from west.commands import WestCommand
 
+from zspdx.version import parse, SUPPORTED_SPDX_VERSIONS, SPDX_VERSION_2_3
 from zspdx.sbom import SBOMConfig, makeSPDX, setupCmakeQuery
```

Comment:
I wondered why `ruff` wasn't complaining about the ordering here, forgot to fix this file 🫣 
If you update the following, we can remove `scripts/west_commands/spdx.py` from the excludes too. Can be done in a follow-up PR too.

```suggestion
from west.commands import WestCommand
from zspdx.sbom import SBOMConfig, makeSPDX, setupCmakeQuery
from zspdx.version import SPDX_VERSION_2_3, SUPPORTED_SPDX_VERSIONS, parse
```